Makefile

```diff
@@ -84,12 +84,13 @@ package_test:
 	make -C targets/ubuntu-12.04
 	make -C targets/ubuntu-14.04
 	make -C targets/ubuntu-16.04
+	make -C targets/ubuntu-18.04
 
 deb-upload:
-	package_cloud push hostedgraphite/$(NAME)/$(DISTRO) out/deb/$(INIT)/$(NAME)_$(VERSION)_$(ARCH).deb
+	package_cloud push hostedgraphite/$(NAME)/$(DISTRO) /tmp/artifacts/out/deb/$(INIT)/$(NAME)_$(VERSION)_$(ARCH).deb
```

Comment:
Not a big fan of seeing `/tmp` here, but I can see what you're doing with the rest of the change.

Reply:
Yep, this was to try keep the change reasonably consistent with all of our other projects that use circle 2.0. I can switch the folder though, not a massive change at all!

Reply:
Nah, no need. Thanks!